### PR TITLE
fixes gibtonite bomb

### PR DESCRIPTION
**Pull request by mc-oofert**
## About The Pull Request

https://github.com/tgstation/tgstation/assets/70376633/8d15d7c9-51c2-4aeb-b1c6-4386168947da

i made it work slightly better too so
also this was in the code since like 2015 but it probably got broken sometime

also 10 second detonation timer is pretty harmless anyway so

## Why It's Good For The Game

bug bad
fyi this was added in 2015 https://github.com/tgstation/tgstation/pull/8587

## Changelog
:cl:
fix: making assembly activated bombs works again
/:cl:

### CODE REVIEW

**Comment by rageguy505** [comment]
i think it'd better if disarming it was done with a wrench, since thats how you disarm singletank bombs.

**Comment by mc-oofert** [comment]
> i think it'd better if disarming it was done with a wrench, since thats how you disarm singletank bombs.

i mean its like a uh electronic thingamajig

**Comment by mc-oofert** [comment]
i should add examine text

**Comment by rageguy505** [comment]
> > i think it'd better if disarming it was done with a wrench, since thats how you disarm singletank bombs.
> 
> i mean its like a uh electronic thingamajig

yeah, and so is the singletank bombs. Im talking slapcrafting that same assembly in your vid onto a internals tank.

**Comment by rageguy505** [comment]
Yeah but it's just more intuitive to disarm two similar bombs the same way

**Comment by ATH1909** [comment]
maybe let both work? although idk what you're wrenching, exactly- it's using screws, not bolts.

**Comment by mc-oofert** [comment]
ok

**Review by Ghommie — APPROVED** [review]

File: code/modules/mining/ores_coins.dm [hunk 1]
```diff
@@ -236,35 +236,77 @@ GLOBAL_LIST_INIT(sand_recipes, list(\
...
+	if(I.tool_behaviour == TOOL_WRENCH && rig)
+		rig.on_found()
+		if(QDELETED(src))
 			return
+		user.balloon_alert_to_viewers("detached rig")
+		user.log_message("detached [rig] from [src].", LOG_GAME)
+		user.put_in_hands(rig)
+		attacher = null
+		cut_overlays(rig_overlay)
+		UnregisterSignal(src, COMSIG_IGNITER_ACTIVATE)
```
Comment (Ghommie) [inline code/modules/mining/ores_coins.dm:308]: Hold on, move these lines of code to `/Exited()`

**Review by Ghommie — CHANGES_REQUESTED** [review]

File: code/modules/mining/ores_coins.dm (hunk 1 above)
Comment (mc-oofert) [reply code/modules/mining/ores_coins.dm]: ok

**Pull request merged**